media-libs/faad2/faad2-2.7.recipe

```diff
@@ -25,6 +23,16 @@ PROVIDES="
 REQUIRES="
 	haiku${secondaryArchSuffix}
 	"
+
+PROVIDES_devel="
+	faad2${secondaryArchSuffix}_devel = $portVersion compat >= 2.0
+	devel:libfaad$secondaryArchSuffix = 2.0.0 compat >= 2
+	devel:libmp4ff$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	faad2$secondaryArchSuffix == $portVersion base
+	"
+
 BUILD_PREREQUIRES="
 	haiku${secondaryArchSuffix}_devel
```

Comment:
Add a BUILD_REQUIRES and move haiku${secondaryArchSuffix}_devel from BUILD_PREREQUIRES to BUILD_REQUIRES.